.env.test

```diff
@@ -0,0 +1,19 @@
+POSTGRES_USER=local_user
```

Comment:
Não precisa duplicar todas as variáveis de ambiente. Pode configurar aqui apenas o que for específico para os testes

Reply:
então... não deveria, mas não consegui ainda adicionar a variável exclusiva desse contexto. `NODE_ENV=test`